Makefile

```diff
@@ -12,7 +12,11 @@ lint:
 unit-test:
 	-docker run -d -p 5432:5432 --name test_db -e POSTGRES_PASSWORD=test_pw postgres
 	PYTHONWARNINGS=ignore:ResourceWarning python3 -m coverage run \
-		-m unittest discover --start-directory tests/unit/backend --top-level-directory . --verbose
+		-m unittest discover --start-directory tests/unit/backend --top-level-directory . --verbose || make clean_test_db
```

Comment:
I'm not sure I follow here. You only want to clean up the test container if python fails?